models/admin.go

```diff
@@ -20,8 +20,10 @@ type NoticeType int
 const (
 	//NoticeRepository type
 	NoticeRepository NoticeType = iota + 1
-	// NoticeTask type
-	NoticeTask
```

Comment:
I would like we have a new column named `status` but not put all in `type`

Reply:
Hmm, I'm not sure whether it's a good idea because it will result in big changs and it's not usefull for ``NoticeRepository `` now.